cmd/gitops/root/cmd.go

```diff
@@ -134,3 +143,12 @@ func configureLogger() {
 		log.SetLevel(log.DebugLevel)
 	}
 }
+
+func initViper(cmd *cobra.Command) error {
+	replacer := strings.NewReplacer("-", "_")
+	viper.SetEnvKeyReplacer(replacer)
+	viper.SetEnvPrefix("GITOPS")
+	viper.AutomaticEnv()
```

Comment:
You don't need to do this bit for each command you register?

Reply:
This is the root command and seems to work for `install` and `ui run` at least so assume it works for others.